.travis.yml

```diff
@@ -11,5 +11,5 @@ jdk:
 deploy:
   provider: bintray
   file: descriptor.json
-  user: yglazko
-  key: a056771a462c6051c1074e7e6791dc951f1da36f
+  user: vaani
+  key: 315213b83b21bc121e7ff476148a598193db810b
```

Comment:
Again, from looking at the example I gave earlier[[1]](http://stackoverflow.com/questions/32356238/issue-using-encrypted-bintray-key-in-travis-yml) this doesn't look to be encrypted.

So, it looks, though I may be wrong, that you placed the vaani password in the open on the net, not a good idea.